README.md

```diff
@@ -63,15 +63,13 @@ molecule = Molecule.from_file(sdf_file_path)
 from openforcefield.topology import Topology
 topology = Topology.from_molecules(molecule)
 
-# Load the smirnoff99Frosst SMIRNOFF force field definition
+# Load the latest OpenFF force field definition
 from openforcefield.typing.engines.smirnoff import ForceField
-forcefield = ForceField('test_forcefields/smirnoff99Frosst.offxml')
+forcefield = ForceField('openff-1.2.0.offxml')
```

Comment:
Updating this should probably be added to the `openforcefields` release process? A function that automatically finds the most recent one comes with some downsides (i.e. `from openforcefield.typing.engines.smirnoff.forcefield import get_available_force_fields, ForceField; sorted([ForceField(file).date for file in get_available_force_fields()])`)

Reply:
[It's already part of the checklist](https://github.com/openforcefield/openforcefields/wiki/Release-Process#7-if-this-is-a-major-release-update-ffs-in-open-force-field-toolkit-examples). We're just always in a rush so we don't do it. 

There's some degree of judgement as to which FF releases are "major" enough to warrant updates to examples. And critically, there is lots of surrounding context -- Like, a bunch of things I fixed in this update had markdown saying "this line loads smirnoff99Frosst", and the actual code loaded Parsley. Further, it required a bit of judgement to know when to use `openff-1.0.0.offxml`, `openff_unconstrained-1.0.0.offxml`, or `openff-1.2.0.offxml`. 

Not sure how to design a process to simplify/automate this. Really the examples contain more information than code, and keeping them maintained is a complex/human intensive process.

Reply:
Ah, well, my suggestion's already covered.

I'd not push for updating the examples more often than a "major" release line (i.e. not updated for Parsley updates, but updated for the bump to Sage). I would not trust the automation I can think of building, and I definitely don't want to do that by hand.